unit-tests-jest/numbers.test.js

```diff
@@ -0,0 +1,43 @@
+import { evenNumbers, toDollars, divideBy, multiplyBy } from './numbers';
+
+describe('evenNumbers', () => {
+  it('returns the even numbers', () => {
+    const numbers = [1, 4, 5, 10, 0];
+    const result = evenNumbers(numbers);
+    expect(result).toEqual([4, 10, 0]);
+  });
+});
+
+describe('toDollars', () => {
+  it('returns number in USD dollar format', () => {
```

Comment:
I think you should add a second test that checks that it properly formats a fractional number, like 28.332.